### ISSUE TITLE
Cannot read property 'addEvent' of null

### ISSUE THREAD

**Issue by rburch** [terminal42/contao-conditionalselectmenu#7]
I keep getting the following error: Uncaught TypeError: Cannot read property 'addEvent' of null at new window.ConditionalSelect (conditionalselect.min.js:14) in console when I go through the checkout with Isotope. I tried turning on moo-tools, I also upgraded conditional select to 3.0.4. I'm using contao 3.5.31 and Isotope 2.4.5.

**Comment by Toflar** [comment]
I guess this happens when the `conditionField`(`Parent Field`) is not correctly selected or not in the DOM? If it's reproducible you should maybe add an issue in Isotope.

**Comment by aschempp** [comment]
Okay, next attempt. Did you possibly disable the country select menu but enable the region selection in your store configuration?

**Comment by rburch** [comment]
I tried with MooTools on but get the same error. It happens in the shipping and review steps. The user can choose State and US or Canada. I tried changing the order of the address fields, but it didn't seem to make a difference. Feel free to add to the cart and go through the process as a guest: https://www.butlersystem.com/carpet-cleaning.html. I am able to checkout completely, so it's not breaking anything.
